src/chrome/CustomIcon.tsx

```diff
@@ -38,7 +40,8 @@ const CustomIcon: React.FunctionComponent<CustomIconProps> = ({
       })}
       width={dimension}
       height={dimension}
-      viewBox="0 0 24 24"
+      transform={`${rotation ? 'rotate('+rotation+')' : ''}`}
```

Comment:
don't use the turnery operator to provide empty values, use `&&` instead:

```suggestion
      transform={rotation && `rotate(${rotation})`}
```